README.md

```diff
@@ -118,35 +118,58 @@ override fun onCreate() {
 <br>
 
 ## Optional Parameters
-You can add optional parameters like userId anytime and update the instance of Sign3Intelligence.
+1.	You can add optional parameters like UserId, Phone Number, etc., at any time and update the instance of Sign3Intelligence.
+2. To update the Sign3Intelligence instance with optional parameters, including additional attributes, you can use the following examples.
+3. Once the options are updated, they get reset. Clients need to explicitly update the options again to ingest them, or else the default value of OTHERS will be sent to the backend.
```

Comment:
default value of OTHERS in userEventType

Reply:
move point 2 to last as examples make sense just after that point

Reply:
Updated all the points.